dependencies/maven/rules.bzl

```diff
@@ -0,0 +1,278 @@
+# Do not edit. deployment_rules_builder.py autogenerates this file from deployment/maven/templates/rules.bzl
+
+#
+# GRAKN.AI - THE KNOWLEDGE GRAPH
+# Copyright (C) 2018 Grakn Labs Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
```

Comment:
Is this license header provided from the template file? What if the user of `graknlabs/deployment` would want to use a different license? For example, we want to use `graknlabs/deployment` rules in `graknlabs/graql` soon, and that repo will be licensed under Apache. Can we make it configurable by the user? This doesn't need to be in this PR.

Reply:
@haikalpribadi 

>Is this license header provided from the template file?

Yes, it is in the template

>What if the user of `graknlabs/deployment` would want to use a different license?

License comment states that user of `graknlabs/deployment` must adhere to a particular license (GNU AGPL v3). If you _don't_ include license header, strictly speaking, the user cannot use your rules [legally]. License header is template does *not* enforce you to have the _same_ license for your project — correct me if I'm wrong here

>This doesn't need to be in this PR.

I will remove it -- but licensing is still an issue

Reply:
What I mean by _"This doesn't need to be in this PR"_ is that any changes to make licensing generic does not need to be in this PR. But the license itself can/should still go in this PR.

Reply:
Well, if a user of `graknlabs/deployment` wants to use it outside of Grakn Labs, it should Apache. I.e. the `graknlabs/deployment` itself will have an Apache License in the root of the repository, and the template files should not contain a license header. Can we make it such that the license header is (optionally) provided by the user through the `genrule` upon generating the file from template?